tests/test_radosbench.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import random
+import sys
+log = logging.getLogger(__name__)
+
+
+def run(**kw):
+    ''' A task for radosbench
+
+    '''
+    log.info("Running radosbench task")
+    ceph_pods = kw.get('ceph_pods')  # list of pod objects of ceph cluster
+    config = kw.get('config')
+
+    clients = []
+    role = config.get('role', 'client')
+
+    clients = [cpod for cpod in ceph_pods if role in cpod.roles]
+
+    idx = config.get('idx', 0)
+    client = clients[idx]
+    pg_num = config.get('pg_num', 64)
+    op = config.get('op', 'write')
+    cleanup = ['--no-cleanup', '--cleanup'][config.get('cleanup', True)]
+    pool = config.get('pool', 'test_pool' + str(random.randint(10, 999)))
+
+    # FIXME: replace pool create with library function
+    pool_create = "ceph osd pool create {pool_name} {pg_num}".format(
+        pool_name=pool,
+        pg_num=pg_num)
+    out, err, ret = client.exec_command(cmd=pool_create,
+                                        check_ec=True)
+    if ret:
+        log.error("Pool creation failed for {}".format(pool))
```

Comment:
Just one hint for formatting with logger, logger itself can easily format message for you, you can just do it like,
```python
log.error("Pool creation failed for %s", pool)
```
We followed this in RHV python code so I think @RazTamir we can follow the same here right?

If ACK from Raz, you can go over all log messages in the code and change it like this.

Reply:
I don't have a hard preference in this case.
Please decide on 1 way and let's stick with it with our code and when reviewing

Reply:
New style string formatting along with old style should be acceptable, Personally new style is what we see in all new repos. https://pyformat.info/

Reply:
Yep, I know about this new style, but what I pointed here that I don't formatting message in advance. I am letting python logger to format the message for me.

Maybe you were thinking about 
```python
log.error("Pool creation failed for %s" % pool)
```
but as you can see I don't using % but just comma and passing variables to the logger itself.

You can see some old discussion here: https://reinout.vanrees.org/weblog/2015/06/05/logging-formatting.html

In python documentation I also all the time see they are letting logger to format message for them:
https://docs.python.org/3/howto/logging-cookbook.html

You can utilize formatter to use { style in instead of % look for documentation here:
https://docs.python.org/3/library/logging.html or https://docs.python.org/3/howto/logging-cookbook.html#use-of-alternative-formatting-styles 
`
If format is specified, use this style for the format string. One of '%', '{' or '$' for printf-style, str.format() or string.Template respectively. Defaults to '%'.
`

So we can stick with {} syntax and still let formatter to format message for us.

If you are formatting message or other strings and storing in some variable we can follow definitely new style {}.format or what @clacroix12  is using f"Format my: {variable}" is also nice if you have already defined variables you would like to use during formatting.

Let's decide on mtg as suggested

Reply:
As @petr-balogh notes, this is about using python logging module, not about string formatting in general.

Reply:
The primary reason why it is preferable to use: `logger.info("some statement: %s %s", arg1, arg2)` is because if `arg1` produces a `TypeError` when formatting, it will not break the app, and `logging` will just capture the exception, log it, and move on. So it is much safer.

Reply:
Followed 'f' strings and considering ease of substitution of variables.